anvil/src/eth/api.rs

```diff
@@ -1476,6 +1477,49 @@ impl EthApi {
         self.backend.load_state(buf).await
     }
 
+    /// Retrieves the Anvil node configuration params.
+    ///
+    /// Handler for RPC call: `anvil_nodeInfo`
+    pub async fn anvil_node_info(&self) -> Result<NodeInfo> {
+        node_info!("anvil_nodeInfo");
+
+        let env = self.backend.env().read();
+        let fork_config = self.backend.get_fork();
+        let tx_order = self.transaction_order.read();
+
+        Ok(NodeInfo {
+            current_block_number: self.backend.best_number(),
+            current_block_timestamp: env.block.timestamp.try_into().unwrap_or(u64::MAX),
+            current_block_hash: self.backend.best_hash(),
+            hard_fork: env.cfg.spec_id,
+            transaction_order: match *tx_order {
+                TransactionOrder::Fifo => "fifo".to_owned(),
+                TransactionOrder::Fees => "fees".to_owned(),
```

Comment:
there's no difference, but is consistent with how we use it

```suggestion
                TransactionOrder::Fees => "fees".to_string(),
```